pages/about.vue

```diff
@@ -32,7 +32,7 @@
                 <div data-testid="member" class="members-list grid"
                     :key="index" v-for="(member, index) in data.members">
                     <MemberComponent :avatarImg="member.avatarImg" :name="member.name" :title="member.title"
-                        :linkedInUrl="member.linkedInUrl" :githubUrl="member.githubUrl" />
+                        :index=index :linkedInUrl="member.linkedInUrl" :githubUrl="member.githubUrl" />
```

Comment:
it's usually recommended to avoid index as the index if possible. Usually I just pick a unique field on the object as the key.

Reply:
I was passing this index down so it can be used in the test in the `data-testid` the way it is used in the `SubmissionListComponent`. I could add an id into the json?